ccd-definition/CaseEventToComplexTypes/children/amendChildren.json

```diff
@@ -346,5 +346,14 @@
     "FieldDisplayOrder": 31,
     "DisplayContext": "READONLY",
     "FieldShowCondition": "party=\"DO_NOT_SHOW\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
does the same need to be done for the addChildren and addRespondents events in the open state? Legal counsellors would never be added at that time so I don't think it would be an issue but might be worth a check